shared-actions/npm-setup/action.yml

```diff
@@ -0,0 +1,31 @@
+name: JS/TS environment setup
+description: 'Set up node & npm config, Get repo and install its dependencies (using cache)'
+inputs:
+  gh-token:
```

Comment:
just to follow the _standard_ name we have, let's have this in capital and underscore:
```suggestion
  GH_TOKEN:
```

Reply:
It is that the inputs seems to be in lower-case by convention:
ie, the same one in an other custom action https://github.com/Typeform/.github/blob/master/shared-actions/secrets-scan/action.yml#L13

Reply:
ok, let's leave it like this then